src/host/enclave.h

```diff
@@ -72,20 +72,20 @@ namespace host
       const EnclaveConfig& enclave_config,
       const CCFConfig& ccf_config,
       std::vector<uint8_t>& node_cert,
-      std::vector<uint8_t>& quote,
       std::vector<uint8_t>& network_cert,
       StartType start_type,
       ConsensusType consensus_type,
       size_t num_worker_thread)
     {
       bool ret;
       size_t node_cert_len = 0;
-      size_t quote_len = 0;
       size_t network_cert_len = 0;
 
       msgpack::sbuffer sbuf;
       msgpack::pack(sbuf, ccf_config);
 
+      std::cout << "Lala" << std::endl;
```

Comment:
Could not find Tinky Winky or Pooh?

Reply:
[I don't see what you are talking about.](https://giphy.com/gifs/teletubbies-po-lala-VCImBo2uUBqqQ)